Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -492,14 +492,14 @@ void addMaterialToGx2fSums(
 /// that we only update the covariance for fitted parameters. (In case of
 /// no qop/time fit)
 ///
-/// @param fullCovariancePredicted The covariance matrix to update
+/// @param fullCovariance The covariance matrix to update
 /// @param aMatrixExtended The matrix containing the coefficients of the linear system.
 /// @param ndfSystem The number of degrees of freedom, determining the size of meaning full block
 ///
 /// @return deltaParams The calculated delta parameters.
-void updateCovariancePredicted(BoundMatrix& fullCovariancePredicted,
-                               Eigen::MatrixXd& aMatrixExtended,
-                               const std::size_t ndfSystem);
+void updateGx2fCovariance(BoundMatrix& fullCovariance,
```

Comment:
I think we need to be explicit that we are updating the covariance of the parameters.
Maybe just `updateGx2fCovarianceParams`?